main.go

```diff
@@ -56,6 +68,11 @@ func main() {
 			Hidden: true,
 			Value:  "error",
 		},
+		&cli.BoolFlag{
+			Name:   "no-color",
+			Hidden: false,
+			Value:  false,
```

Comment:
Good point.

BTW urfave/cli contains bug. The position of global flag like `--no-color` is not relaxed. You have to use it after main command (not after sub-command). It means, that you can use this:

```
rhc --no-color status
```

but you cannot use this, because it is terminated with error of incorrect usage:

```
rhc status --no-color
```

There is issue about this: https://github.com/urfave/cli/issues/1113

Reply:
BTW: the cli package tries to parse boolean value of environment variable in this case. If the environment variable is "1", "true", "True", "0", "false", "False", then it works as expected. When environment variable is anything else, then the rhc will be terminated with following error:

```
could not parse "OK" as bool value for flag no-color: strconv.ParseBool: parsing "OK": invalid syntax
```

Reply:
Yea. I'm okay with that. Those are the values accepted by ParseBool, and it seems reasonable to follow the format set by that function. If we wanted "OK" to be a valid boolean-like, we should submit that upstream in the strconv package. NO_COLOR is a convention, so I can't say this definitively, but it sounds like a boolean variable, so only accepting boolean-like strings is fine with me. Setting NO_COLOR=blue doesn't make sense, so exiting with an error is reasonable.

Reply:
I don't think that "OK" should be considered as valid boolean value. It was just example.